pandas/tests/extension/test_numpy.py

```diff
@@ -218,6 +218,13 @@ def test_check_dtype(self, data, request):
             )
         super().test_check_dtype(data)
 
+    def test_is_not_object_type(self, dtype, request):
+        if dtype.numpy_dtype == "object":
+            request.node.add_marker(
+                pytest.mark.xfail(reason="PandasDtype(object) should be object")
+            )
```

Comment:
Can an xfail be avoided here?

Reply:
For the same reasoning in https://github.com/pandas-dev/pandas/pull/49378/files#r1009898830, I think this should unexpectedly (and understandably) fail